tools/common_lib/src/memory_bandwidth.h

```diff
@@ -32,11 +32,12 @@ class MemoryBandwidthDispatcher : public NodeDispatcher
         }
     };
 public:
-    MemoryBandwidthDispatcher(create_params_t&& params, ID3D12Device* d3d12_device, ID3D12GraphicsCommandList* cmd_list, IntelExtension& intc_ext)
+    MemoryBandwidthDispatcher(create_params_t&& params, ID3D12Device* d3d12_device, ID3D12GraphicsCommandList* cmd_list, IntelExtension& intc_ext, bool use_stateless)
```

Comment:
For MemoryBandwidthDispatcher we don't need to use stateless and we can keep descriptor path. Please remove it.
(just for simplicity)

Reply:
removed